src/Sentry/Extensibility/DisabledHub.cs

```diff
@@ -147,6 +147,20 @@ public bool CaptureEnvelope(Envelope envelope)
     /// </summary>
     public SentryId CaptureEvent(SentryEvent evt, Scope? scope = null, SentryHint? hint = null) => SentryId.Empty;
 
+    /// <summary>
+    /// No-Op.
+    /// </summary>
+    public void CaptureFeedback(SentryFeedback feedback, Action<Scope> configureScope)
+    {
+    }
+
+    /// <summary>
+    /// No-Op.
+    /// </summary>
+    public void CaptureFeedback(SentryFeedback feedback, SentryHint? hint, Action<Scope> configureScope)
```

Comment:
I know this is prob the pattern we have now across our APIs, but ideally `hint` would be the last parameter. This when when calling you don't have to pick one when going from `feedback` to `feedback, scopeCallback`.

Reply:
Good call. We could have one overload instead of two (if hint has a default value of null) 👍🏻